spec/models/query/results_sums_integration_spec.rb

```diff
@@ -130,7 +130,7 @@
     create(:float_wp_custom_field)
   end
   let(:type) do
-    create(:type).tap do |t|
+    create(:type) do |t|
```

Comment:
Is it this kind of things that triggered you to create the Rubocop pull request?

Reply:
Yes :)